src/js/dataMapping/search/awardType.js

```diff
@@ -10,6 +10,18 @@ export const awardTypeCodes = {
     'B': 'Purchase Orders (PO)',
     'C': 'Delivery Orders (DO)',
     'D': 'Definitive Contracts',
+    'E': 'Unknown Type',
+    'F': 'Cooperative Agreement',
+    'G': 'Grant for Research',
+    'S': 'Funded Space Act Agreement',
+    'T': 'Training Grant',
+    'IDV_A': `Government-Wide Acquisition Contract`,
```

Comment:
These can be in single quotes '' rather than back-ticks `` because they are not template literals.